client_code/MultiSelectDropDown/__init__.py

```diff
@@ -233,6 +264,10 @@ def _mk_popover(self, init_node, **event_args):
         # this is a bit of a hack - we're using the libraries private methods for this
         init_node(self._el.data("selectpicker")["$bsContainer"])
 
+    def form_show(self, **event_args):
+        """This method is called when the HTML panel is shown on the screen"""
+        self.width = self.width
+
```

Comment:
This was based on my original implementation using just JQuery. Since the original property setter is before the dom node is actually present on the page, I just use a dummy property setting trigger in the form_show event. It doesn't have anything to do with the default, but I have added "fit-content" in _defaults because that's a good idea :)